### ISSUE TITLE
hardcopy for tests for students giving error

### ISSUE THREAD

**Issue by pstaabp** [openwebwork/webwork2#1930]
I was testing #1809 and found an error, but this is in develop, so I don't think it is related to that PR. 

If you go to a test and on the overview page, click the hardcopy download button, you're taken to the Hardcopy page.  Clicking generate Hardcopy results in
```
Can't call method "open_date" on an undefined value at /opt/webwork/webwork2/lib/WeBWorK/ContentGenerator/Hardcopy.pm line 224.
```

**Comment by somiaj** [comment]
I can confirm this issue while testing as well.

**Comment by drgrice1** [comment]
Fixed by #1931.
